components/Buttons/src/ColorThemer/MDCButtonColorThemer.m

```diff
@@ -50,6 +50,11 @@ + (void)resetUIControlStatesForButtonTheming:(nonnull MDCButton *)button {
   }
 }
 
++ (void)applySemanticColorScheme:(nonnull id<MDCColorScheming>)colorScheme
```

Comment:
Some branch issues. @romoore  To respond to your outstanding comment in the previous PR:
"Why do we have a specific method for toRaisedButton: if there is no such class and it just calls the general themer?"

The thought behind it is that `MDCRaisedButton` is a subclass of `MDCButton` just like `MDCFlatButton`. As a subclass that adds customization to `MDCButton`,  having its own themer will allow design iterations on `MDCRaisedButton` without having to create new theming API. 

Also, for consistency sake it felt more right to have developers be aware that just like `MDCFlatButton` has a themer, so does `MDCRaisedButton`, and doesn't need the developer to make assumptions on which button has its own themer and which doesn't.

Reply:
Then should we have an MDCRaisedButtonColorThemer? In general should all color theming go into a single class for a component?

Reply:
That's a great question. I don't have a clear answer to what our collective preference is in that case. I usually tend to collate if it makes logical sense. What do you think?

Reply:
Since we don't break-up our components into smaller targets, it probably makes no difference if there is one or two classes for theming. 

On a related note, I see we've mixed the `MDCColorScheme` and `MDCColorScheming` themers in Feature Highlight.  I think my preference would be to at least separate the two themers since they depend on different scheme classes.

Reply:
The general pattern for themers has been to have one themer for a given subsystem per component, with multiple APIs for different themer implementations.